### PR TITLE
Add useful PortForwardGet to cliutils

### PR DESCRIPTION
**Pull request by ashleywang1**
I'm using it in meshctl, and this function is super useful!

### CODE REVIEW

**Comment by arianaw66** [comment]
Should we remove from Gloo while we're at it and have that call this instead to avoid duplicate code?

File: cliutils/port_forward.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+func PortForwardGet(ctx context.Context, namespace string, resource string, localPort string, kubePort string, verbose bool, getPath string) (string, *exec.Cmd, error) {
+
+	/** port-forward command **/
+
+	portFwd, err := PortForward(namespace, resource, localPort, kubePort, verbose)
+	if err != nil {
+		return "", nil, err
+	}
+
+	localCtx, cancel := context.WithTimeout(ctx, time.Second*30)
```
Comment (arianaw66) [inline cliutils/port_forward.go:49]: This change from 3 sec to 30 sec is intentional, right?

**Comment by ashleywang1** [comment]
> Should we remove from Gloo while we're at it and have that call this instead to avoid duplicate code?

I'm not sure! This would require an upgrade of go-utils, which could break something.

File: cliutils/port_forward.go (hunk 1 above)
Comment (ashleywang1) [reply cliutils/port_forward.go:49]: Nope!

Comment (arianaw66) [reply cliutils/port_forward.go:49]: Oh sorry. I think I was looking at the PR, and someone probably changed it since.

**Review by arianaw66 — APPROVED** [review]

**Pull request merged**